### ISSUE TITLE
Validation accuracy more than train

### ISSUE THREAD

**Issue by UsamaI000** [attilanagy234/neural-punctuator#8]
Hi, I am getting validation accuracy greater than training and validation loss lesser than train. Can you tell why? Maybe due to class imbalance? I am using different dataset

**Comment by bana513** [comment]
Hi, validation loss can be lower than train, e.g. using dropout during training. Best practice is to the model with early stopping on validation loss. We cannot say anything about your accuracy, especially if you are using different dataset. However, we recommend using F-score instead of accuracy with imbalanced datasets.
